chrome/content/zotero/xpcom/itemTreeView.js

```diff
@@ -260,6 +261,44 @@ Zotero.ItemTreeView.prototype.setTree = Zotero.Promise.coroutine(function* (tree
 });
 
 
+Zotero.ItemTreeView.prototype.setSortColumn = function() {
+	var colId, col, dir, currentCol, currentDir;
```

Comment:
Doesn't really matter with a small function like this, but as a general style rule, I'd say this is fine for `col`, `currentCol`, and `currentDir`, since those might not be set, but `colId` and `dir`, which are always assigned at top level, should have `var` on their respective lines.